lightning/src/offers/invoice.rs

```diff
@@ -380,6 +380,20 @@ macro_rules! invoice_builder_methods { (
 	}
 } }
 
+#[cfg(test)]
+macro_rules! invoice_builder_methods_test { (
+	$self: ident, $self_type: ty, $invoice_fields: expr, $return_type: ty, $return_value: expr
+	$(, $self_mut: tt)?
```

Comment:
I'm still checking that these inscrutable arguments make sense for the new macro, but it seemed good enough for a test-changes-only PR...